config/project_config.py

```diff
@@ -57,11 +55,12 @@ class ProjectConfig(ProjectConfigDTO):
     Project Config implementation
     """
 
-    def __init__(self, config_path: Path) -> None:
+    def __init__(self, project_root: Path, config_path: Path) -> None:
```

Comment:
@WhiteJaeger will we need these changes in the official distribution of project_config? it is shared across all our projects